src/main/java/org/spongepowered/common/config/category/OptimizationCategory.java

```diff
@@ -119,6 +119,11 @@
             "suppress the exceptions printing out in the log.")
     private boolean disableFailingAdvancementDeserialization = true;
 
+    @Setting(value = "disable-pathfinding-chunk-loads", comment =
+            "In vanilla, pathfinding may result in chunks being loaded, This provides a method to disable chunks from loading during pathing."
```

Comment:
Wrap the comment to 60 chars (I think?) as above. Make it a little more explicit as to what the configuration option does, for example (probably needs tweaking further):

```
"In vanilla, pathfinding may result in loading chunks.\n
You can disable that here, which may result in a\n
performance improvement. This may not work well\n
with mods."
```